ubuntu/common/check_uvm_service.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+
+if test -f "/usr/local/cuda/bin/nvcc"; then
+	
+   sudo mv check_nvidia_uvm.sh /usr/local/bin/check_nvidia_uvm.sh
+   
+   echo -e '[Unit]\n\nDescription=Runs /usr/local/bin/check_nvidia_uvm.sh\n\n' \
+           | sudo tee nvidia_uvm_startup.service
+   echo -e '[Service]\n\nExecStart=/usr/local/bin/check_nvidia_uvm.sh\n\n' \
+           | sudo tee -a nvidia_uvm_startup.service
+   echo -e '[Install]\n\nWantedBy=multi-user.target' \
+   	| sudo tee -a nvidia_uvm_startup.service
+   
```

Comment:
The EOF syntax is useful, and can be used here when working with multi-line text in Bash

Reply:
Good idea! I rewrote this with the EOF syntax and it's much more readable now.